test/invoice-workflow.e2e-spec.ts

```
@@ -0,0 +1,884 @@
+      const taskId = startResponse.body.taskId;
+      console.log(`✅ Workflow started with task ID: ${taskId}`);
+
+      console.log('🔄 Monitoring workflow lifecycle...');
+
+      let lifecycleCompleted = false;
+      const maxMonitoringAttempts = 15;
+
+      for (let i = 0; i < maxMonitoringAttempts; i++) {
+        await TestUtils.wait(300);
+
+        const tasksResponse = await request(app.getHttpServer())
```

ESLint

[error] 'tasksResponse' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)